README.md

```diff
@@ -1,7 +1,6 @@
-# PRE-ALPHA STAGE WARNING
+# ALPHA STAGE
```

Comment:
We are beyond Alpha, I recommend we mark this as Preview Stage.

Reply:
Done.